themes/openy_themes/openy_rose/templates/page/html.html.twig

```diff
@@ -30,6 +30,7 @@
     node_type ? 'page-node-type-' ~ node_type|clean_class,
     db_offline ? 'db-offline',
     node ? 'node-id-' ~ node.id,
+    theme_name
```

Comment:
There is no need to add php logic here, just add `openy_rose` string.
According to the existed workflow, active theme could be named not as openy_rose, end user creating sub themes that could have another name.